msteams-platform/publishing/office-store-checklist.md

```diff
@@ -24,39 +23,37 @@ The follow metadata is required in your manifest.json file and for AppSource Sel
 ## Localized content
 
 > [!NOTE]
-> Microsoft Teams plans to support localized content for the following metadata. Currently, only English is supported for Teams apps.
+> AppSource plans to support localized content for the following metadata. Currently, only English is supported for Teams apps, but in preparation you can provide additional languages for these. See [localizing your app](~/publishing/apps-localization.md) for more information.
 
 |Data|Type|Size|Manifest|Seller Dashboard|Description|
 |---|---|---|---|---|---|
 |App name|String|30|`name.short`|✔|The name for your application as it should appear in the storefront and in product.|
 |Long app name|String|30|`name.full`|✔|The name for your application as it should appear in the storefront and in product.|
 |Short description|String|80|`description.short`|✔|Short description of your app.|
 |Long description|String|4000|`description.full`|✔|A more detailed description of your app. In the manifest file, an accurate summary is adequate. In the Seller Dashboard, you can use a richer and formatted description for AppSource product page.|
-|Screen shots (1&ndash;5)|.png, .jpg, or .gif|1366w &times; 768h and smaller than 1024 KB||✔|At least one screen shot that shows your app experience. Uses on the app details page.|
+|Screen shots (1-5)|.png, .jpg, or .gif|1366w x 768h and smaller than 1024 KB||✔|At least one screen shot that shows your app experience. Uses on the app details page.|
 
 ## Submission extras for bots
 
-Bots in Microsoft Teams must be created using Bot Framework. See [Create a bot](~/concepts/bots/bots-create) for instructions. Use your 96&times;96 color icon for your bot's icon in Bot Framework.
+Bots in Microsoft Teams must be created using Bot Framework. See [Create a bot](~/concepts/bots/bots-create.md) for instructions. Use your 96x96 color icon for your bot's icon in Bot Framework.
 
 ## Screenshots
 
- The screenshots that are uploaded on the Seller Dashboard will eventually be displayed on Microsoft AppSource to provide a visual preview of your app/product along with your app description.
+The screenshots that are uploaded on the Seller Dashboard are displayed on [Microsoft AppSource](https://appsource.microsoft.com/marketplace/apps?product=office%3Bteams&page=1) to provide a visual preview of your app along with your app description.
```

Comment:
..."are displayed in both Microsoft AppSource and your app listing in the Teams Client"...